### ISSUE TITLE
Crashes when opening the same file in two split panes

### ISSUE THREAD

**Issue by loganhuskins** [atom/atom#10303]
When I try to open the same file in a split window on both sides, the application stops responding and I have to force a restart. I can reproduce this behavior consistently in the application. The files are JavaScript, if that matters.

OS X 10.11.2
Atom 1.3.2

All plugins are disabled.

[Atom_2016-01-06-073202_Logans-MacBook-Pro.txt](https://github.com/atom/atom/files/79847/Atom_2016-01-06-073202_Logans-MacBook-Pro.txt)

**Comment by chrishoffman** [comment]
I am running into a similar issue that I think is related.

In my case, I have two project folders loaded with identical folder stuctures.
```
❯ tree projectA
projectA
├── folder1
│   └── file.txt
└── folder2
    └── file.txt

❯ tree projectB
projectB
├── folder1
│   └── file.txt
└── folder2
    └── file.txt
```

Here are the steps to reproduce:
- In projectA, open `folder1/file.txt`.  Tab name is `file.txt`.
- In projectA, open `folder2/file.txt`.  Tab names are now `folder1/file.txt` and `folder2/file.txt`, respectively.
- In projectB, open `folder1/file.txt`.  Atom hangs and must be force restarted.

**Comment by klerk** [comment]
I have the same issue than @chrishoffman on windows 7 and atom 1.3.2

**Comment by 50Wliu** [comment]
This has already been fixed in the 1.4 betas.
